src/Tribe/Main.php

```diff
@@ -2642,6 +2642,7 @@ public function dateToTimeStamp( $date, $hour, $minute, $meridian ) {
 		 * @todo remove - unused
 		 */
 		protected function dateHelper( $date ) {
+			_deprecated_function( __FILE__, '3.11', 'date' );
```

Comment:
note my @todo -- this function is not used anywhere in our code and can actually just be removed. i don't think we need to deprecate class methods.